Options.py

```diff
@@ -260,6 +407,16 @@ def from_text(cls, text: str) -> Range:
                 return cls(random.randint(cls.range_start, cls.range_end))
             else:
                 raise Exception(f"random text \"{text}\" did not resolve to a recognized pattern. Acceptable values are: random, random-high, random-middle, random-low, random-range-low-<min>-<max>, random-range-middle-<min>-<max>, random-range-high-<min>-<max>, or random-range-<min>-<max>.")
+        elif text in {"on", "true", "default", "yes", "high"}:
```

Comment:
I don't like adding all these names, I think just "low", "high", "default" and maybe "middle" (to reflect the random ranges) is plenty sufficient.

Reply:
The motivation for this was for changing a setting from a toggle to a range.

Even if we don't end up changing the progression balancing from toggle to range, there may be something else in the future that makes that change.

If someone has a yaml file they have been using for a long time with `progression_balancing: on`, and that changes to a range, do you want to give them an error and force them to change that to something else, when we could just give them the default?

Reply:
Then the Prog Balancing subclass can have those additions, or a "transition" subclass, no reason to "pollute" every range.